src/ipfsd-daemon.js

```diff
@@ -136,8 +136,8 @@ class Daemon {
     // in case we decide to change it at
     // the daemon level in the future
     if (bits) {
-      args.concat(['-b', bits])
-      log(`initializing with keysize: ${bits}`)
+      args.push('-b')
+      args.push(bits)
```

Comment:
Can we log the args array here - something like:

 ```
log(`initializing with args: ${args.join(' ')}`)
``` 

That should help expose issues like this.

Reply:
`exec` does command line logging.  That's how I found the error.

Reply:
oh wow. We never got a speed improv then? How did we shave 5 mins the last time?